cinema/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255, unique=True)
+    description = models.CharField(max_length=255)
+    duration = models.IntegerField()
+
+    class Meta:
+        verbose_name_plural = "movies"
```

Comment:
It's not nessesary to use verbose_name_plural here because in this case Django will do it correct itself